apps/src/sites/studio/pages/components/LevelTokenDetails.jsx

```diff
@@ -0,0 +1,153 @@
+/* global levelKeyList */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
```

Comment:
I followed the pattern in https://github.com/code-dot-org/code-dot-org/blob/d344587164a1671898b827852c72e0e892918d79/apps/src/publicKeyCryptography/IntegerDropdown.jsx#L6-L8.

Is there a better way to do this?